Remove contribute and wiki from header, replace with tag line

critzo opened the pull request:
This PR addresses issue #164 by removing the Contribute and Wiki links and images in the header, and per our discussions in staff meeting, replaces it with a tagline linking to the /who/ page.

This PR can be previewed on my fork:
https://critzo.github.io/m-lab.github.io/

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/m-lab/m-lab.github.io/212)
<!-- Reviewable:end -->

georgiamoon commented:
*[_data/header-footer-content.yml, line 6 \[r1\]](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/212#-KMg1443U8t-U6m8Ytte:-KMg1444xbCgYJgx0H07:204768243) ([raw file](https://github.com/m-lab/m-lab.github.io/blob/e9d3cc7b2773f799d0d44e48371fbbcdd8f09ad6/_data/header-footer-content.yml#L6)):*
> ```YAML
>   ctas:
>   -
>     link: "/who/"
> ```

What do you think about this going to the about page? then we don't have to feel like we need to update the who page. We could also think about "who" being a portion of the about page longer term.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/212)*
<!-- Sent from Reviewable.io -->

georgiamoon commented:
*[_data/header-footer-content.yml, line 7 \[r1\]](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/212#-KMg1veXhk1za-Kl7_29:-KMg1veXhk1za-Kl7_2A:1443673238) ([raw file](https://github.com/m-lab/m-lab.github.io/blob/e9d3cc7b2773f799d0d44e48371fbbcdd8f09ad6/_data/header-footer-content.yml#L7)):*
> ```YAML
>   -
>     link: "/who/"
>     h2: "A partnership between New America's Open Technology Institute, Google Open Source Research, Princeton University's PlanetLab, and other supporting partners. "
> ```

Would it be easy to add links for OTI and PLC to the text here?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/212)*
<!-- Sent from Reviewable.io -->

georgiamoon commented:
*[_data/header-footer-content.yml, line 7 \[r1\]](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/212#-KMg1veXhk1za-Kl7_29:-KMg2H7fGfZXSF-J73AL:-240459759) ([raw file](https://github.com/m-lab/m-lab.github.io/blob/e9d3cc7b2773f799d0d44e48371fbbcdd8f09ad6/_data/header-footer-content.yml#L7)):*
<details><summary><i>Previously, georgiamoon (georgia bullen) wrote…</i></summary>
> Would it be easy to add links for OTI and PLC to the text here?
</details>
I would also rephrase just slightly to add: "Measurement Lab is ..."

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/212)*
<!-- Sent from Reviewable.io -->

sloane101 commented:
I agree with Georgia's suggestion to start with "Measurement Lab is..." Elsewhere, Measurement Lab is referred to as a consortium - I don't have strong feelings about whether you use partnership or consortium, but just flagging it in case you want to keep that terminology consistent with what is in your about section.

critzo commented:
@georgiamoon @sloane101 I've incorporated your suggestions to the extent possible. There may be a way to add links to the text, but adding them just as html didn't work- it messed up the formatting of the "learn more" link, so I just updated the text and the main link to point at the about page. Allowing html in the header text will likely require looking at how the site is generated or at least on how that section gets parsed from YAML to html. In any case, this PR is ready for review and merge.

critzo commented:
@georgiamoon I updated the template file to use the 8 column width css as we discussed. You can preview on my fork.

georgiamoon commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---







*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/m-lab.github.io/212#-:-KNmivgT1P9nB6iozaaV:1417088721)*
<!-- Sent from Reviewable.io -->

georgiamoon commented:
LGTM.
